kubernetes/zenko/charts/backbeat/values.yaml

```diff
@@ -96,7 +96,7 @@ ingestion:
 
 lifecycle:
   conductor:
-    cronRule: "0 */6 * * *"
+    cronRule: 0 */6 * * *
```

Comment:
yaml can correctly read this? damn

Reply:
Typically yaml parsers read almost everything as a string. The exceptions are Ints and Bools.